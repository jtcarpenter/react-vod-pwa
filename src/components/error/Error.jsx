import React from 'react';

export function Error({errorMessage}) {

    return (
        <div className="error">
            {errorMessage}
        </div>
    );
}

export default Error;