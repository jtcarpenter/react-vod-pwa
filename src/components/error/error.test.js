import React from 'react';
import {shallow} from 'enzyme';
import renderer from 'react-test-renderer';
import {Error} from 'components/error/Error';

const mockError = (
    <Error errorMessage="test error"/>
)

describe('Error', () => {
    it('should render', () => {
        const wrapper = shallow(mockError);
        expect(wrapper.exists()).toBe(true);
    })

    it('should display an error message', () => {
        const error = renderer.create(mockError);
        expect(error.toJSON()).toMatchSnapshot();
    });
});