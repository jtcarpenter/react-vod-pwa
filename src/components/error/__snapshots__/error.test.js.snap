// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should display an error message 1`] = `
<div
  className="sc-bdVaJa BPjXy"
>
  test error
</div>
`;
